### PR TITLE
add quantity of sellers condition

### PR DESCRIPTION
**Pull request by IncognitaDev**
#### What problem is this solving?

This change add a new condition to product page, `isSellersMoreThan`, this condition allows to pass a quantity of sellers minimum to show the layout.

#### How to test it?

[Workspace](https://luis--iaqua.myvtex.com/produto-teste-unico/p)

#### Screenshots or example usage:
1 seller: 
image

More than 1 seller: 
image

### CODE REVIEW

File: react/ConditionLayoutProduct.tsx [hunk 1]
```diff
@@ -81,6 +82,17 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
...
+  isSellersMoreThan({ values, args }) {
+    const { sellers } = values
+
+    const productAvailable = sellers?.filter(seller => (
+      seller.commertialOffer.AvailableQuantity > 0
+    ))
+
+    const isMoreThen = sellers?.length > args?.quantity && productAvailable.length > args?.quantity
+
+    return Boolean(isMoreThen)
```
Comment (igorbrasileiro) [inline react/ConditionLayoutProduct.tsx:94]: This already is a boolean, right?  Maybe won't need this constructor

File: react/ConditionLayoutProduct.tsx [hunk 2]
```diff
@@ -81,6 +82,17 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
 
     return Boolean(isAvailable)
   },
+  isSellersMoreThan({ values, args }) {
```
Comment (igorbrasileiro) [inline react/ConditionLayoutProduct.tsx:85]: Lint this file

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Good job @IncognitaDev ! Just a small request change

File: react/ConditionLayoutProduct.tsx [hunk 3]
```diff
@@ -81,6 +82,19 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
...
+
+    const productAvailable = sellers?.filter(
+      (seller) => seller.commertialOffer.AvailableQuantity > 0
+    )
+
+    const isMoreThen =
+      sellers?.length > args?.quantity &&
+      productAvailable.length > args?.quantity
+
+    return isMoreThen
```
Comment (lariciamota) [inline react/ConditionLayoutProduct.tsx:96]:
```suggestion
    const isMoreThan =
      sellers?.length > args?.quantity &&
      productAvailable.length > args?.quantity

    return isMoreThan
```

**Review by lariciamota — CHANGES_REQUESTED** [review]
Nice! Thank you for the contribution

**Review by igorbrasileiro — APPROVED** [review]

**Review by lariciamota — APPROVED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -102,6 +102,7 @@ Possible values for the `subject` prop:
 | `specificationProperties`  | List of product specifications currently displayed on the UI. | `{ name: string, value: string }`. Notice: `value` is an optional prop. If omitted, only the specification name (`name`) will be checked. |
 | `areAllVariationsSelected` | Whether all product variations currently available on the UI were selected by the user (`true`) or not (`false`). | No arguments are expected. |
 | `isProductAvailable`                  | Whether the product is available (`true`) or not (`false`).  | No arguments are expected. |
+| `isSellersMoreThan`                  | Whether the quantity of seller is more than argument passed.  | `{ quantity: number }`|
```
Comment (victorhmp) [inline docs/README.md:105]: How about `hasMoreSellersThan`?

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 4 above)
Comment (victorhmp) [reply docs/README.md:105]: @lariciamota and @igorbrasileiro what do you think?

Comment (lariciamota) [reply docs/README.md:105]: That sounds better! 🚀

Comment (IncognitaDev) [reply docs/README.md:105]: realy sounds better, I couldn't think of a better name.

File: docs/README.md [hunk 5]
```diff
@@ -102,6 +102,7 @@ Possible values for the `subject` prop:
 | `specificationProperties`  | List of product specifications currently displayed on the UI. | `{ name: string, value: string }`. Notice: `value` is an optional prop. If omitted, only the specification name (`name`) will be checked. |
 | `areAllVariationsSelected` | Whether all product variations currently available on the UI were selected by the user (`true`) or not (`false`). | No arguments are expected. |
 | `isProductAvailable`                  | Whether the product is available (`true`) or not (`false`).  | No arguments are expected. |
+| `hasMoreSellersThan`                  | Whether the quantity of seller is more than argument passed.  | `{ quantity: number }`|
```
Comment (victorhmp) [inline docs/README.md:105]:
```suggestion
| `hasMoreSellersThan`                  | Whether the quantity of sellers for the product is more than argument passed.  | `{ quantity: number }`|
```

File: react/ConditionLayoutProduct.tsx [hunk 6]
```diff
@@ -81,6 +82,19 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
...
+  hasMoreSellersThan({ values, args }) {
+    const { sellers } = values
+
+    const productAvailable = sellers?.filter(
+      (seller) => seller.commertialOffer.AvailableQuantity > 0
+    )
+
+    const isMoreThan =
+      sellers?.length > args?.quantity &&
+      productAvailable.length > args?.quantity
```
Comment (victorhmp) [inline react/ConditionLayoutProduct.tsx:94]: `productAvailable` could be `undefined` here, right? If so the call to `productAvailable.length` would break

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: react/ConditionLayoutProduct.tsx (hunk 6 above)
Comment (icazevedo) [reply react/ConditionLayoutProduct.tsx:94]: This needs to be fixed

Comment (icazevedo) [reply react/ConditionLayoutProduct.tsx:94]: Also, how come `sellers.length` be lesser than `productAvailable.length`? I think this is a redundant check

**Review by icazevedo — CHANGES_REQUESTED** [review]

**Review by icazevedo — APPROVED** [review]

**Review by victorhmp — APPROVED** [review]

**Pull request merged**